### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by ForHemer**
In this project, I will add some functionality to the application to make it interactive.

### CODE REVIEW

File: src/modules/addRemove.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Task from './task.js';
+import taskCompleted from './removeCompleted.js';
 
 class displayTask {
   static displayTasks() {
```
Comment (oloomoses) [inline src/modules/addRemove.js:5]: - [x] Kindly ensure that a task remains checked when the browser is reloaded. This means that you need to update the localStorage when a task is checked.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @ForHemer, 

**Highlights**
- Highly organized/clean code
- README
- UI Design

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addRemove.js (hunk 1 above)
Comment (ForHemer) [reply src/modules/addRemove.js:5]: @oloomoses The required change has been implemented

**Review by Kingstalux — APPROVED** [review]
Hi @ForHemer, 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**